step.yml

```diff
@@ -27,18 +27,18 @@ description: |-
   - [Generate Cordova Build Configuration](https://www.bitrise.io/integrations/steps/android-build)
   - [Run npm command](https://www.bitrise.io/integrations/steps/npm)
   - [Cordova Archive](https://www.bitrise.io/integrations/steps/cordova-archive)
-  
+
 website: https://github.com/bitrise-steplib/steps-ionic-archive
 source_code_url: https://github.com/bitrise-steplib/steps-ionic-archive
 support_url: https://github.com/bitrise-steplib/steps-ionic-archive/issues
 project_type_tags:
-- ionic
+  - ionic
 type_tags:
-- build
```

Comment:
I think these are unintended changes (our yml standard is to start the list with the same indentation as the parent, right?)

Reply:
I'm using GoLand default YML indentation settings. Is there a place where we defined how do we indent yml? 🤔

Reply:
In the devcenter samples, they are not indented indeed.